packages/ckeditor5-link/ckeditor5-metadata.json

```
@@ -55,7 +55,10 @@
 			"path": "src/linkimage.js",
 			"requires": [
 				"Link",
-				"Image"
+				[
+					"ImageBlock",
+					"ImageInline"
+				]
```

```suggestion
				"ImageBlock"
```

While playing with the plugins and editors, the editor throws an error while bootstrapping with `ImageInline` and `LinkImage`.

```
CKEditorError: plugincollection-soft-required {"missingPlugin":"ImageBlockEditing","requiredBy":"LinkImageUI"}
```

Hence, `LinkImage` must load `ImageBlock`.